### PR TITLE
Update plot.py to include sharex, sharey

### PR DESCRIPTION
**Pull request by v715**
Adresses #290

### CODE REVIEW

**Comment by v715** [comment]
Docstrings for `sharex` and `sharey` taken directly from the matplotlib documentation https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.subplots.html

File: wfdb/plot/plot.py [hunk 1]
```diff
@@ -78,6 +79,12 @@ def plot_items(signal=None, ann_samp=None, ann_sym=None, fs=None,
         also be set to 'all' for all channels. Major grids at 0.5mV, and minor
         grids at 0.125mV. All channels to be plotted with grids must have
         `sig_units` equal to 'uV', 'mV', or 'V'.
+	sharex, sharey : bool or {'none', 'all', 'row', 'col'}, default: False
```
Comment (Lucas-Mc) [inline wfdb/plot/plot.py:82]: Can you follow the same format as the rest of the docstrings for consistency?

File: wfdb/plot/plot.py [hunk 2]
```diff
@@ -210,21 +217,23 @@ def create_figure(n_subplots, figsize):
         The number of subplots to generate.
     figsize : tuple
         The figure's width, height in inches. 
+	sharex, sharey : bool or {'none', 'all', 'row', 'col'}, default: False
```
Comment (Lucas-Mc) [inline wfdb/plot/plot.py:220]: Same here... consistent format for docstrings

File: wfdb/plot/plot.py [hunk 3]
```diff
@@ -210,21 +217,23 @@ def create_figure(n_subplots, figsize):
...
-
     """
-    fig = plt.figure(figsize=figsize)
-    axes = []
-
-    for i in range(n_subplots):
-        axes.append(fig.add_subplot(n_subplots, 1, i+1))
-
+	fig, axes = plt.subplots(
+		nrows=n_subplots, ncols=1, sharex=sharex, sharey=sharey, figsize=figsize
```
Comment (Lucas-Mc) [inline wfdb/plot/plot.py:235]: Something is wrong here I think.. a tab instead of spaces? It's causing the tests to fail

**Review by Lucas-Mc — CHANGES_REQUESTED** [review]
Thanks for this! This is something I've been wanting to do after trying to plot 29 signals at once! There are still some residual issues but I can merge after those are fixed

File: wfdb/plot/plot.py (hunk 3 above)
Comment (v715) [reply wfdb/plot/plot.py]: this is what i get for editing code directly on github instead of locally cloning the repo 😁

**Comment by v715** [comment]
hey lucas, made the changes. looks like all tests have passed. lmk if there's anything else u want changed.

**Pull request merged**